homeassistant/components/zwave_js/climate.py

```diff
@@ -192,6 +192,9 @@ def _current_mode_setpoint_enums(self) -> List[Optional[ThermostatSetpointType]]
         if self._current_mode is None:
             # Thermostat(valve) with no support for setting a mode is considered heating-only
             return [ThermostatSetpointType.HEATING]
+        if self._current_mode.value is None:
+            # guard missing value
+            return []
```

Comment:
Do we need to handle empty list when accessing this property?

Reply:
I wasn't sure so took the safe route. Can it handle a None value being returned?

Reply:
I mean it looks like we expect the list property to have at least one item. Here we return an empty list. Will that work?

Reply:
I've moved the guard to the place where this value is requested.